### ISSUE TITLE
What is the meaning of labels?

### ISSUE THREAD

**Issue by zhouhao-learning** [markovmodel/deeptime#25]
I see your data_generator function, but `labels = np. empty ((input_data [0]. shape [0], 2 * output_size). Astype ('float32')`, which is not real labels, is there any point in defining labels like this? So what is my model learning?
Can you give me some specific explanations? Thank you very much.

**Comment by Obudo** [comment]
The labels are required by Keras loss function format. We use empty labels as VAMPnets are a case of unsupervised learning since the loss function can be evaluated using only the training data.

**Comment by zhouhao-learning** [comment]
@Obudo 
Well, thank you very much!
Another question. What does `lag` mean in the paper?

**Comment by amardt** [comment]
Hey,
lag is another word for the $\tau$ timestep from the paper. It means how many timesteps you want to predict in the future,  broadly speaking. 
You will always train a model to approximate the time propagator K for a specific timestep. Therefore, you split your data into $traj(t)$ and $traj(t+\tau)$. The first trajectory is then $\tau$ lagging behind, so to say.

Hope this makes it clearer!
Best
Andreas

**Comment by zhouhao-learning** [comment]
@amardt 
OK, thank you for your guidance.
